### PR TITLE
feat: add --fast option to run backend integration tests only 

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: None

Signed-off-by: Fabio Tranchitella <fabio@tranchitella.eu>

### CODE REVIEW

File: main_comment.go [hunk 1]
```diff
@@ -188,10 +191,12 @@ func parsePrOptions(commentBody string) (map[string]string, error) {
 							" somerepo/pull/12/head --pr somerepo/1.0.x ",
 					)
 				}
-				prRepos[userInputParts[0]] = revision
+				buildOptions.PullRequests[userInputParts[0]] = revision
 			}
+		} else if word == "--fast" {
+			buildOptions.Fast = true
```
Comment (merlin-northern) [inline main_comment.go:197]: general comment: it needs an entry in the online help: https://github.com/mendersoftware/integration-test-runner/blob/master/main_pullrequest.go#L135

Comment (tranchitella) [reply main_comment.go:197]: https://github.com/mendersoftware/integration-test-runner/pull/175/commits/71bf5f3576d59083a7633666d275a1ab68f6168d

**Review by MaciejTe — APPROVED** [review]
Looks good!

File: model.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package main
+
+type BuildOptions struct {
```
Comment (MaciejTe) [inline model.go:3]: :thumbsup:

**Pull request merged**